slamon_afm/app.py

```diff
@@ -1,21 +1,46 @@
 import logging
+
+import os
+
 from flask import Flask
 
 from slamon_afm.models import db
 from slamon_afm.routes import agent_routes, bpms_routes, status_routes, dashboard_routes
 
 
+def _log_level_from_str(log_level):
+    try:
+        return int(log_level)
+    except ValueError:
+        numeric_level = getattr(logging, log_level.upper(), None)
+        if not isinstance(numeric_level, int):
+            raise ValueError('Invalid log level: %s' % log_level)
+        return numeric_level
+
+
+def _bool_from_str(value):
+    if value.lower() in ['true', 'yes']:
```

Comment:
This was exactly the motivation behind this one. I mean, validating the input instead of silently falling back to `False`. The dictionary solution would achieve this in a neat way, but still I would stick with the more verbose exception.